trestle/transforms/transformer_helper.py

```diff
@@ -42,31 +44,47 @@ class PropertyAccounting():
     def __init__(self) -> None:
         """Initialize."""
         self._group_map: Dict[str, int] = {}
-        self._property_map: Dict[str, Dict[str:int]] = {}
+        self._property_map: Dict[str, Dict[str, int]] = {}
 
-    def count_group(self, group: str = None) -> None:
+    def count_group(self, group: Optional[str] = None) -> None:
         """Group accounting."""
+        if not group:
+            raise TrestleError('count_group created with group=None')
         if group not in self._group_map:
             self._group_map[group] = 0
         self._group_map[group] += 1
 
     def count_property(
-        self, group: str = None, name: str = None, value: str = None, class_: str = None, ns: str = None
+        self,
+        group: Optional[str] = None,
+        name: Optional[str] = None,
+        value: Optional[str] = None,
+        class_: Optional[str] = None,
+        ns: Optional[str] = None
     ) -> None:
         """Property accounting."""
         key = _segment_separator.join([str(name), str(value), str(class_), str(ns)])
+        if not group:
+            raise TrestleError('count_property created with group=None')
         if group not in self._property_map:
             self._property_map[group] = {}
         if key not in self._property_map[group]:
             self._property_map[group][key] = 0
         self._property_map[group][key] += 1
 
     def is_common_property(
-        self, group: str = None, name: str = None, value: str = None, class_: str = None, ns: str = None
+        self,
+        group: Optional[str] = None,
+        name: Optional[str] = None,
+        value: Optional[str] = None,
+        class_: Optional[str] = None,
+        ns: Optional[str] = None
     ) -> bool:
         """Check for common property."""
         rval = False
         key = _segment_separator.join([str(name), str(value), str(class_), str(ns)])
+        if not group:
```

Comment:
If it is optional I think this should not raise an error, otherwise it shouldn't be optional

Reply:
Removed optional and the default val of None - and removed error.